src/wordpress/config.py

```diff
@@ -65,23 +78,48 @@ def keep_wp_sites(dir_name):
                     yield WPResult(wp_config.wp_site.path, "KO", "", "", "", "", "")
 
     def run_wp_cli(self, command):
+        """
+        Execute a WP-CLI command. The command doesn't have to start with 'wp '. It will be added automatically, and
+        it's the same for --path option.
+
+        Argument keywords:
+        command -- WP-CLI command to execute
+        """
         cmd = "wp {} --path='{}'".format(command, self.wp_site.path)
         return Utils.run_command(cmd)
 
     @property
     def is_installed(self):
+        """
+        Check if a WordPress is installed
+
+        Return:
+        True, False
+        """
         # checkt that index.php has been created
         valid_path = os.path.join(self.wp_site.path, 'index.php')
         return os.path.exists(valid_path)
 
     @property
     def is_config_valid(self):
+        """
+        Tells if WordPress configuration is valid
```

Comment:
i.e connection to DB successfull with credentials from wp-config.php

Reply:
daune